deepchem/feat/materials_featurizers.py

```diff
@@ -0,0 +1,285 @@
+"""
+Featurizers for inorganic crystals.
+"""
+
+import numpy as np
+
+from deepchem.feat import Featurizer
+from deepchem.utils import pad_array
+
+
+class ChemicalFingerprint(Featurizer):
```

Comment:
@nd-02110114 Great point about the need to modify `Featurizer` I have an upstream PR (part of #1792) that does this. I'll try to get this merged in sooner :)

+1 for the idea of `ElementPropertyFingerprint` instead of `ChemicalFingerprint`.